src/new-client/src/app/lib/types/index.tsx

```diff
@@ -126,8 +127,116 @@ type Session = {
   services: string[];
 };
 
+type FetchAvailabilityRequest = {
+  sites: string[];
+  service: string;
+  from: string;
+  until: string;
+  queryType: '*' | 'Days' | 'Hours' | 'Slots';
+};
+
+type AvailabilityResponse = {
+  site: string;
+  service: string;
+  availability: Availability[];
+};
+
+type Availability = {
+  date: Date;
+  blocks: AvailabilityBlock[];
+};
+
+type AvailabilityBlock = {
+  from: string;
+  until: string;
+  count: number;
+};
+
+// TODO: Rename this type to something more meaningful
+type Week = {
+  start: number;
+  end: number;
+  startMonth: number;
+  endMonth: number;
+  startYear: number;
+  endYear: number;
+  unbooked?: number;
+  totalAppointments?: number;
+  booked?: number;
+  bookedAppointments: BookedAppointments[];
+  startDate?: dayjs.Dayjs;
+  endDate?: dayjs.Dayjs;
+};
+
+type FetchBookingsRequest = {
+  from: string;
+  to: string;
+  site: string;
+};
+
+type Booking = {
+  reference: string;
+  from: string;
+  duration: number;
+  service: string;
+  site: string;
+  status: AppointmentStatus;
+  attendeeDetails: AttendeeDetails;
+  contactDetails?: ContactItem[];
+  reminderSet: boolean;
+  created: string;
+  // TODO: Additional data object - any type?
+};
+
+type AttendeeDetails = {
+  nhsNumber: string;
+  firstName: string;
+  lastName: string;
+  dateOfBirth: Date;
+};
+
+type ContactItem = {
+  type: ContactItemType;
+  value: string;
+};
+
+type BookedAppointments = {
+  service: string;
+  count: number;
+};
+
+type ClinicalService = {
+  label: string;
+  value: string;
+};
+
+enum AppointmentStatus {
+  Unknown,
+  Provisional,
+  Booked,
+  Cancelled,
+}
+
+enum ContactItemType {
+  Phone,
+  Email,
+  Landline,
+}
+
 // TODO: Decide where this info should live and move it there
-const clinicalServices = [{ label: 'RSV (Adult)', value: 'RSV:Adult' }];
+const clinicalServices: ClinicalService[] = [
+  { label: 'COVID 12-15', value: 'COVID:12_15' },
+  { label: 'COVID 16-17', value: 'COVID:16_17' },
+  { label: 'COVID 18-74', value: 'COVID:18_74' },
+  { label: 'COVID 5-11 (10 Min)', value: 'COVID:5_11_10' },
+  { label: 'COVID 75+', value: 'COVID:75' },
+  { label: 'FLU 18-64', value: 'FLU:18_64' },
+  { label: 'FLU 65+', value: 'FLU:65' },
+  { label: 'FLU and COVID 18-64', value: 'COVID_FLU:18_64' },
+  { label: 'FLU and COVID 65-74', value: 'COVID_FLU:65-74' },
+  { label: 'FLU and COVID 75+', value: 'COVID_FLU:75' },
+  { label: 'RSV (Adult)', value: 'RSV:Adult' },
+];
```

Comment:
Ooh I really should've spotted this before, apologies. 
This will have a knock-on effect on the Create Availability Wizard, because the list of services you're allowed to select in `select-services-step.tsx` just maps over this array. 
Can we quickly check with the team whether this is fine, or whether we need to filtered it down to just RSV for the beta?